draft-ietf-mls-protocol.md

```diff
@@ -2447,26 +2436,48 @@ and administered under a Specification Required policy {{!RFC8126}}.
 
 ## MLS Ciphersuites
 
-The "MLS Ciphersuites" registry lists identifiers for suites of cryptographic
-algorithms defined for use with MLS.  These are two-byte values, so the maximum
-possible value is 0xFFFF = 65535.  Values in the range 0xF000 - 0xFFFF are
-reserved for vendor-internal usage.
+A cipher suite value defines a combinaison of a protocol version
+and of the set of cryptographic algorithms used for this version MLS.
```

Comment:
A ciphersuite is a combination of a protocol version and the set of cryptographic algorithms that should be used.

Reply:
Thanks, will address this in another commit.